### PR TITLE
fix(tabs): can't scroll when vertical mode in safari

### LINKED ISSUE #29964: Tab组件垂直布局在safari下无法滑动
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://ant.design/components/tabs-cn/#components-tabs-demo-slide](https://ant.design/components/tabs-cn/#components-tabs-demo-slide)

### Steps to reproduce
1. 在滑动demo中切换至vertical
2. tabs无法滑动，且子tab高度有问题

### What is expected?
 tabs可滑动，子tab高度自适应内部组件

### What is actually happening?
tabs无法滑动，且子tab高度有问题

| Environment | Info |
|---|---|
| antd | 4.15.0 |
| React | 16.1 |
| System | macos Catalina 10.15.7 |
| Browser | Safari浏览器 13.1.3 |

### PR DESCRIPTION
**Pull request by kerm1it**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

fix https://github.com/ant-design/ant-design/issues/29964

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Fix Tabs can't scroll when vertical mode in Safari 13.1.         |
| 🇨🇳 Chinese |  修复 Tabs 在 Safari 13.1 垂直模式下不能滚动问题。         |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/tabs/style/index.less [hunk 1]
```diff
@@ -47,6 +47,7 @@
     .@{tab-prefix-cls}-nav-list {
       position: relative;
       display: flex;
+      flex: 1 0 auto; // fix safari scrool problem
```
Comment (yoyo837) [inline components/tabs/style/index.less:50]: 注释typo

Comment (yoyo837) [reply components/tabs/style/index.less:50]:
```suggestion
      flex: 1 0 auto; // fix safari scroll problem
```

**Review by shaodahong — APPROVED** [review]
LGTM

**Comment by shaodahong** [comment]
rc-tabs 的也有问题，可以顺便修复下

**Comment by kerm1it** [comment]
还漏了一个，下面的 ... 按钮也没有显示出来

**Pull request merged**